### PR TITLE
Nandini

### PR DESCRIPTION
**Pull request by Nandiniverma2024**

### CODE REVIEW

File: src/atoms/headings/one/large/large-white/index.astro [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1 class="text-5xl md:text-8xl lg:text-9xl text-white font-medium font-merriweather">
+    <slot/>
+</h1>
```
Comment (devdotmohit) [inline src/atoms/headings/one/large/large-white/index.astro:3]: @Nandiniverma2024 rename the directory from headings/one/large/.. to headings/one/white/index.astro. Add a new line at the end of the file.

File: src/components/common/header/index.astro [hunk 2]
```diff
@@ -4,10 +4,55 @@
...
 ---
-<header class="bg-black h-12">
+<header class="bg-black h-20 fixed z-50 w-full items-center">
     <Container>
-        <section class="flex items-center h-full">
-            <p class="text-white">Header</p>
+        <section class="flex items-center justify-between cursor-pointer">
+            <div class="flex">
+                <div class="h-18 w-40 py-4">
+                    <img src="./public/logo/Lightfield-logo.png" alt="">
```
Comment (devdotmohit) [inline src/components/common/header/index.astro]: @Nandiniverma2024 add a class with properties h-full and w-full to this image tag. Import path should be /logo/.. and not ./public.

File: src/components/common/header/index.astro [hunk 3]
```diff
@@ -4,10 +4,55 @@
...
+                </div>
+                <div>
+                    <img src="./public/logo/Lightfield-logo.png" alt="" class="h-18 w-40 py-5 md:hidden" onclick="dropdown()">
+                    <ul class="md:flex md:bg-black md:space-x-6 md:mr-20 md:py-6 md:space-y-0 md:px-0 py-5 pl-8 pr-20 text-base space-y-4 bg-white right-0 fixed z-50" id="submenu">
```
Comment (devdotmohit) [inline src/components/common/header/index.astro]: @Nandiniverma2024 why is bg-black inside md? Is it something specific to the screen resolution?

File: src/components/home/hero/index.astro [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+    subtitle
+
+} = Astro.props
+
 ---
-<section>Hero</section>
+<section class="h-auto w-full">
+    <div class="relative">
+        <div class="h-full w-full">
+            <img src="./public/images/hero-section.jpg" alt="" class="z-9 bg-cover">
```
Comment (devdotmohit) [inline src/components/home/hero/index.astro]: @Nandiniverma2024 Import image error.

**Review by devdotmohit — CHANGES_REQUESTED** [review]
@Nandiniverma2024 Make these changes and we will review this again.

File: src/components/common/header/index.astro (hunk 3 above)
Comment (Nandiniverma2024) [reply src/components/common/header/index.astro]: For small screen sizes, we have a white background for menu options

**Pull request merged**